modules/appeals_api/app/swagger/appealable_issues/v0/swagger.json

```diff
@@ -340,35 +340,45 @@
       },
       "productionOauth": {
         "type": "oauth2",
-        "description": "This API uses OAuth 2 with the authorization code grant flow. [More info](https://developer.va.gov/explore/authorization?api=claims)",
+        "description": "The authentication model for the Appealable Issues API uses OAuth 2.0/OpenID Connect. The following authorization models are supported: [Authorization code flow](https://developer.va.gov/explore/api/appealable-issues/authorization-code) and [Client Credentials Grant (CCG)](https://developer.va.gov/explore/api/appealable-issues/client-credentials).",
         "flows": {
           "authorizationCode": {
-            "authorizationUrl": "https://api.va.gov/oauth2/authorization",
-            "tokenUrl": "https://api.va.gov/oauth2/token",
+            "authorizationUrl": "https://api.va.gov/oauth2/appeals/v1/authorization",
+            "tokenUrl": "https://api.va.gov/oauth2/appeals/v1/token",
             "scopes": {
               "veteran/AppealableIssues.read": "Appealable issues info",
+              "veteran/appeals.read": "Appeals info",
               "representative/AppealableIssues.read": "Appealable issues info",
+              "representative/appeals.read": "Appeals info"
+            }
+          },
+          "clientCredentials": {
+            "tokenUrl": "To get production access, you must either work for VA or have specific VA agreements in place. If you have questions, [contact us](https://developer.va.gov/support/contact-us).",
```

Comment:
I struggled a bit with how to model the production OAuth `tokenUrl`. It seemed wrong to put a description in a field meant for a URL, but we also don't publish this URL for production, only for sandbox (see below). I took this wording directly from the [developer portal CCG pages](https://staging-developer.va.gov/explore/api/higher-level-reviews/client-credentials).